docs/apidiffs/current_vs_latest/opentelemetry-sdk-trace.txt

```diff
@@ -1,4 +1,12 @@
 Comparing source compatibility of  against 
+***! MODIFIED INTERFACE: PUBLIC ABSTRACT io.opentelemetry.sdk.trace.samplers.SamplingResult  (not serializable)
+	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
+	+++! NEW METHOD: PUBLIC(+) STATIC(+) io.opentelemetry.sdk.trace.samplers.SamplingResult drop()
+	+++! NEW METHOD: PUBLIC(+) STATIC(+) io.opentelemetry.sdk.trace.samplers.SamplingResult drop(io.opentelemetry.api.common.Attributes)
```

Comment:
we'd still like to get rid of these overloads that have attributes as a parameter. Thanks.

Reply:
![image](https://user-images.githubusercontent.com/44219737/124218314-d8b77b00-daae-11eb-95ce-dfcec1806c54.png)
I think this kind of call case is also necessary.

But I will try my best to think about how to replace these calls with attributes as parameter.

Reply:
Based on all the above, I feel that we should keep the old calls with attributes, and use the new short method in the future development process. I have achieved this idea and hope you approve.